seatunnel-apis/seatunnel-api-spark/src/main/scala/org/apache/seatunnel/spark/BaseSparkTransform.scala

```diff
@@ -14,12 +14,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.github.interestinglab.waterdrop.spark
+package org.apache.seatunnel.spark
 
 import org.apache.spark.sql.{Dataset, Row}
-
-import io.github.interestinglab.waterdrop.apis.BaseTransform
-import io.github.interestinglab.waterdrop.config.{Config, ConfigFactory}
+import org.apache.seatunnel.config.ConfigFactory
+import org.apache.seatunnel.apis.BaseTransform
+import org.apache.seatunnel.config.{Config, ConfigFactory}
```

Comment:
`import org.apache.seatunnel.config.ConfigFactory` Duplicate package is introduced, please check

Reply:
![019F96AF](https://user-images.githubusercontent.com/95013770/147210900-cadac756-98fd-4197-b2fe-0c8611178c9c.png)

Reply:
Good catch! I add a9bd31b to remove it. Thanks 👍

Reply:
@zhongjiajie I think there are still some that have not been modified, please check the overall, thank you

Reply:
hi @zhongjiajie, some files in `.github` folder are also modified.

or, we can modify them later, because most of the project has been modified, and this patch is not suitable for staying for a long time.